src/content/public goods funding.md

```diff
@@ -0,0 +1,287 @@
+---
+title: Public goods funding 
+description: An overview of public goods funding solutions on Ethereum
+lang: en
+template: use-cases
+emoji: 🌎
+sidebar: true
+sidebarDepth: 2
+summaryPoint1: Public goods are nonexcludable and nonrivalrous and benefit everyone
+summaryPoint2: Traditional models of funding public goods are inefficient and encourage coordination failures
+summaryPoint3: Public goods markets on Ethereum make it easier to fund and develop critical digital public goods
```

Comment:
Is 'Public good markets' something you seen elsewhere? It isn't vocabulary I've come across anywhere else.

Reply:
Yeah, I got the idea from [this post](https://forum.effectivealtruism.org/posts/ZjBnzzqkrB7JNLjTg/the-case-for-promoting-creating-public-goods-markets-as-a) on the Effective Altruism forum. I hadn't come across that word before reading the article, but the rationale made sense enough to include it (there's a detailed explanation of the concept in another section).

Reply:
Changed to: "Ethereum provides useful infrastructure for improving funding and development of critical public goods." Does that work for you?